packages/node/src/indexer/indexer.manager.ts

```diff
@@ -325,6 +323,15 @@ export class IndexerManager {
 
     // blockOffset and genesisHash should only have been created once, never updated.
     // If blockOffset is changed, will require re-index and re-sync poi.
+
+    // Project is new and requries a starting block
+    if (!project && !keyValue.lastProcessedHeight) {
```

Comment:
As we discussed, for a new project, mark the startBlock-1 as processed is weird since it is against the truth.